mm/2s2h/Enhancements/Player/FasterSceneTransitions.cpp

```diff
@@ -0,0 +1,54 @@
+#include <libultraship/bridge.h>
```

Comment:
We recently moved away from importing the LUS bridge header as its a vary large header and increases compile times. This can be swapped for -- to include the minimum for what you are using
```suggestion
#include "public/bridge/consolevariablebridge.h"
```